bot/package.json

```diff
@@ -1,34 +1,37 @@
 {
   "name": "airhornbot",
   "version": "1.0.0",
+  "description": "",
   "main": "index.js",
-  "author": "Discord",
+  "author": "Sky W",
```

Comment:
As much as we appreciate the contribution here, I do think we would need to keep the author as Discord

Reply:
This and the change about the `"private": true` in package.json were from me starting the code from a fresh start. (Typed my name in while doing `npm init`, oops.)